src/main/cms/cms_menu_misc.c

```diff
@@ -88,7 +88,7 @@ static const void *cmsx_menuRcOnDisplayUpdate(displayPort_t *pDisp, const OSD_En
     UNUSED(selected);
 
     for (int i = 0; i <= AUX4; i++) {
-        rcDataInt[i] = lroundf(rcData[i]);
+        rcDataInt[i] = lrintf(rcData[i]);
```

Comment:
This changes the only occurrence of `lroundf` which saves space since we won't need to include this function.
Only half-way `.5` cases are affected: while `lrintf` round to nearest even number, `lroundf` rounds away from zero. I think it's fine to also use `lrintf` in this case?

Reply:
These values are just displayed in the OSD menu so this should be fine. Actually I think `lrintf()` is better here because it rounds to nearest instead of away from zero. Rounding away from zero will give you a deadband of [ -1, 1 ] around zero. Not ideal for displaying the values IMO.